spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-dependencies-parent</artifactId>
-		<version>2.0.0.M6</version>
+		<version>2.0.0.RC1</version>
```

Comment:
This is still wrong. There is no such a version of this `spring-cloud-dependencies-parent`. Isn't it?